app/serializers/current_user_serializer.rb

```diff
@@ -288,6 +289,10 @@ def include_associated_account_ids?
     SiteSetting.include_associated_account_ids
   end
 
+  def needs_required_fields_check?
```

Comment:
As we’re not processing the value of `#needs_required_fields_check?`, I don’t think we need this method. `attributes :needs_required_fields_check?` should already do the job if I’m not mistaken.

Reply:
![](https://media3.giphy.com/media/v1.Y2lkPTc5MGI3NjExaWw4MjE3bTFjbnc5M3NtYWpqNmdoNW52djl5bG1pejQ3NHl0cHM5ayZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/3oFzmkkwfOGlzZ0gxi/giphy.webp)